modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -6621,6 +6622,80 @@ public void testPatchObjectEntryWithKeywords() throws Exception {
 		Assert.assertEquals("tag3", keywordsJSONArray.get(2));
 	}
 
+	@Test
+	public void testPatchObjectEntryWithRelationship() throws Exception {
```

Comment:
@p-albuquerque please check if there is a existing test that allows us to reuse the logic

Reply:
I believe I can do something into [testPartialUpdateObjectEntry](https://github.com/liferay/liferay-portal/blob/d1809559499892538038ccf8c5bc19dece28ac36/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java#L3362-L3483)